tests/functional/simple_selection/test_graph_select.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+import os
+from dbt.tests.util import run_dbt, run_sql_file
+from dbt.tests.tables import TableComparison, get_tables_in_schema
+
+# steps happened:
+# 1. copy all of the sql statement over in python file
+# 2. moved seed.sql over
+# 3. select * from {{ source('raw', 'seed') }} has to be updated to {{ this.schema }}.seed
+# 4. created models become created tables, assertFalse might need to be updated
+# 5. assert_correct_schemas needs to be implemented
+
+# Question how to validate the tests actually correctly tested the intended case
+
+# idea of script to automate this:
+# 1. unittest2pytest can handle convert assert,
+# 2. probably need something like the steps described above to convert the sql files
+# 3. another pass to fix the functions and report remaining functions
+# run_sql_file means need to add create table function
```

Comment:
@gshank  what's your thoughts on this whole section

Reply:
We don't need assertFalse. Just use 'assert' and a condition that evaluates how we need. We do not want to update the source('raw', 'seed') because that's a built-in dbt feature. If the 'seed' command is executed, the seed table should be there and that SQL should work. If unittest2pytest helps, that's great. Put something in the CONVERTING.md file describing it.

Reply:
assert_correct_schema isn't used in any other tests that I can see, so putting it in this test case seems fine.

Reply:
Just fyi we generally should not usually have to update any schema references, because the original tests all used randomly generated schema names. This particular test did some config changes to the schema name in the SQL (a legitimate thing to test...) so it's special that way.

Reply:
For the source('raw', 'seed') part, I will get an error saying the table doesn't exist If I don't update it. What's the proper way to update it?